packages/ui/InfoTile/index.tsx

```diff
@@ -0,0 +1,18 @@
+"use client"
+
+import {InfoTileStyled, InfoValueWrapper} from './infoTile.styled'
+
+export type InfoTileProps = {
+  label: string;
+  children: string | JSX.Element;
```

Comment:
you could change it like below, this way it includes both string and components (basically anything what React can render) -> https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/basic_type_example/#useful-react-prop-type-examples

```suggestion
  children: React.ReactNode;
```

Reply:
@PawelPazderski instead of rendering `children`, try passing a component to render as a prop named `value` as the story specified (though admittedly the Jira story could use some polish). Zuza's `NavItem` inside `NavList` accepts `componentToRender` as a prop - this is the solution we are looking for here. 
I know it doesn't make that much difference in the end (component will work and look the same), but it's a good learning experience regardless.